File: Web/src/app/models/class.section.model.ts
export class ClassSectionModel {
    ClassId: string;
    ClassName: string;
    SectionId: string;
    SectionName: string;

    constructor(studentItem?: any) {
        studentItem = studentItem || {};

        this.ClassId = studentItem.ClassId || "";
        this.ClassName = studentItem.ClassName || "";
        this.SectionId = studentItem.SectionId || "";
        this.SectionName = studentItem.SectionName || "";
    }
}

File: Services/Igmite.Lighthouse.Models/ClassSectionModel.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Runtime.Serialization;

namespace Igmite.Lighthouse.Models
{
    [DataContract, Serializable]
    public class ClassSectionModel
    {
        public ClassSectionModel()
        {
        }

        [Key, DataMember]
        public Guid VTId { get; set; }

        [Key, DataMember]
        public Guid ClassId { get; set; }

        [DataMember]
        public string ClassName { get; set; }

        [Key, DataMember]
        public Guid SectionId { get; set; }

        [DataMember]
        public string SectionName { get; set; }
    }
}